docs/config.rst

```diff
@@ -360,7 +367,7 @@ Complete list of settings that you can put into ``testenv*`` sections:
 
 .. conf:: changedir ^ PATH ^ {toxinidir}
 
-    Change to this working directory when executing the test command.
+    Change to the current working directory when executing the test command.
```

Comment:
```suggestion
    Change the current working directory when executing the test command.
```

The original change could be confused for "this command changes the workind directory be the current working directory".